pages/common/secrethub.md

```diff
@@ -0,0 +1,24 @@
+# secrethub
+
+> A tool to keep secrets out of config files.
+> More information: <https://secrethub.io>.
+
+- Print a secret to stdout:
+
+`secrethub read {{path/to/secret}}`
+
+- Generate a random value and store it as a new secret (version):
```

Comment:
What does the word (version) in the parentheses mean ? It's not very clear to me.

Reply:
Hi @agnivade, great feedback! :smile:

When writing to a path where a secret is already stored, a new version of that secret is created.

When reading a secret, it's possible to read a specific version with: `{{path/to/secret}}:{{version}}`, e.g. `path/to/secret:2`. When the version is omitted (as in the first example of this tldr page), it defaults to `path/to/secret:latest`, which retrieves the latest version of the secret.

It's a challenge to keep it concise while also explaining it enough :smile: I've opened https://github.com/tldr-pages/tldr/pull/3847 as an attempt to make this clearer. What do you think? More feedback is welcome :+1:

Reply:
Yes, the new PR looks good to me. Thanks !